setup.py

```diff
@@ -8,20 +8,28 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+def load_version():
     """Returns the version found in nilearn/version.py
 
     Importing nilearn is not an option because there may dependencies
     like nibabel which are not installed and setup.py is supposed to
     install them.
     """
-    locals_dict = {}
+    # load all vars into globals, otherwise
+    #   the later function call using global vars doesn't work.
     globals_dict = {}
     with open(os.path.join('nilearn', 'version.py')) as fp:
-        exec(fp.read(), globals_dict, locals_dict)
+        exec(fp.read(), globals_dict)
 
-    return locals_dict['__version__']
+    return globals_dict
 
+
+def is_installing():
+    # Allow command-lines such as "python setup.py build install"
+    return len(sys.argv) > 1 and 'install' in sys.argv
```

Comment:
> The exact same mechanism exists in distutils:

Great!

> Note that setuptools is actually used in nilearn, this is why I suggested the
> previous one.

Shit. I missed that. AFAICT it was added by @lesteve in decembre.

This is really a no-no. Using setuptools makes installs be eggs, which
circumvent the normal Python import mechanisms and has many detrimental
consequences [*]. We should not be using setuptools. Besides, this adds a
dependency (setuptools is not in the standard library).

[*] One of them, amongst others:
http://gael-varoquaux.info/programming/long-syspath-and-consequences-one-more-reason-not-to-use-easy_install.html